piplicenses.py

```diff
@@ -242,23 +242,30 @@ def get_pkg_info(pkg):
             pkg_info['license_classifier'],
             pkg_info['license'])
 
-        if fail_on_licenses and license_name in fail_on_licenses:
-            sys.stderr.write("fail-on license {} was found for package "
-                             "{}:{}".format(
-                                license_name,
-                                pkg_info['name'],
-                                pkg_info['version'])
-                             )
-            sys.exit(1)
-
-        if allow_only_licenses and license_name not in allow_only_licenses:
-            sys.stderr.write("license {} not in allow-only licenses was found"
-                             " for package {}:{}".format(
-                                license_name,
-                                pkg_info['name'],
-                                pkg_info['version'])
-                             )
-            sys.exit(1)
+        license_names = set(license_name.split(', '))
```

Comment:
Thanks for your P-R.

I have one concern.

Multiple licenses are separated by `, ` because that is how pip-licenses behaves when it searches and finds them.

There are some "license name" in the world that contain `, `.

See also)
* [SPDX License List | Software Package Data Exchange (SPDX)](https://spdx.org/licenses/)
* [Classifiers · PyPI](https://pypi.org/classifiers/)

What do you think?

Your implementation using `Set.difference()` is simple and elegant. I think there is a benefit to merging these.

Reply:
I would also like comments from recent contributors if they have any.

ping @dkarp0 @cdce8p

Reply:
@raimon49 I wasn't aware of licenses containing a comma. To tackle that,  what we can do is use the ; delimiter to concatenate multiple licenses.